### DISCUSSION TITLE
Caching of album artwork thunmnails

### DISCUSSION THREAD

**Discussion by dusnm** [jeffvli/sonixd#182] [Ideas]
Currently, every time a view is rendered the service pulls album artwork off of the subsonic server. This makes the application appear sluggish. You could add a caching feature that'll enable you to pull album artworks off disk instead through the network.

**Comment by jeffvli** [comment]
There's already image caching in place, although there are a few things I may need to adjust.

Which views seem slow to you?

**Reply by dusnm** [reply]
All of them. It's very noticeable. All the thumbnails take a while to fill in. Keep in mind my machine is quite powerful (Ryzen 5 3600) and the subsonic server is in the same local network on a Gigabit Ethernet connection. Either the caching layer isn't working properly or isn't very well optimized.

**Reply by jeffvli** [reply]
If possible, could you do a screen capture of what it looks like for you? Even with the current implementation it shouldn't be excessively slow especially if the server is on your LAN. 

I just removed my custom caching solution in favor of electron's so the images should be loaded from its local cache so hopefully it's better in the next release in #192.
